### PR TITLE
Add the Code of Conduct and fix hydration issues due to markdown link references

### PR DESCRIPTION
**Pull request by benjagm**
**GitHub Issue:** #278 
Fixes: #278 

**Summary**: Add the Code of Conduct and fix hydration issues due to markdown link references
Changes included:
* Adding the CoC through submodule + symlink
* Fixed hydration issues due to markdown link references

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

No

### CODE REVIEW

**Comment by benjagm** [comment]
Feedback provided in Slack. Great work... just some additional changes pending.

File: pages/overview/code-of-conduct/index.page.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+export async function getStaticProps() {
+  const block1 = fs.readFileSync('pages/overview/code-of-conduct/_index.md', 'utf-8')
+  const { content: block1Content } = matter(block1)
+  return {
+    props: {
+      blocks: [block1Content]
+    }
+  }
+}
```
Comment (Relequestual) [inline pages/overview/code-of-conduct/index.page.tsx:17]:
```suggestion
export async function getStaticProps() {
  const block = fs.readFileSync('pages/overview/code-of-conduct/_index.md', 'utf-8')
  const { content: blockContent } = matter(block)
  return {
    props: {
      blocks: [blockContent]
    }
  }
}
```

File: pages/overview/code-of-conduct/index.page.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <SectionContext.Provider value='docs'>
+      <Head>
+        <title>{newTitle}</title>
+      </Head>
+      <StyledMarkdown markdown={blocks[0]} />
+    </SectionContext.Provider>
+  )
+}
+ContentExample.getLayout = getLayout
```
Comment (Relequestual) [inline pages/overview/code-of-conduct/index.page.tsx:31]:
```suggestion
export default function Content ({ blocks }: { blocks: any[] }) {
  const newTitle = 'Code of Conduct'

  return (
    <SectionContext.Provider value='docs'>
      <Head>
        <title>{newTitle}</title>
      </Head>
      <StyledMarkdown markdown={blocks[0]} />
    </SectionContext.Provider>
  )
}
Content.getLayout = getLayout
```

**Review by Relequestual — COMMENTED** [review]
A few naming suggestions but otherwise looks good and produces the required result. Thanks.

**Pull request merged**